src/types/library-types.ts

```diff
@@ -0,0 +1,30 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
```

Comment:
Should this be a .d.ts file instead

Reply:
Maybe?? I guess since this is just declaring types, not adding actual functionality, it can be?

Reply:
pushed